src/actions/home.js

```diff
@@ -0,0 +1,11 @@
+import * as constants from '../constants/home';
+import {home} from '../data/index.json';
```

Comment:
could you use `data.json` instead of `data/index.json`?

Reply:
have you applied prettier here?

Reply:
Yes it is necessary. I will update in the next commit